components/eamxx/src/physics/shoc/eamxx_shoc_process_interface.hpp

```diff
@@ -158,11 +158,6 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
       wprtp_sfc(i)  = surf_evap(i)/rrho_i(i,nlevi_v)[nlevi_p];
       upwp_sfc(i) = surf_mom_flux(i,0)/rrho_i(i,nlevi_v)[nlevi_p];
       vpwp_sfc(i) = surf_mom_flux(i,1)/rrho_i(i,nlevi_v)[nlevi_p];
-
-      const int num_qtracer_packs = ekat::npack<Spack>(num_qtracers);
```

Comment:
We can prob remove num_qtracers from the class itself (and from the set_variable call) as well.

Reply:
Done.